src/builders/transform.ts

```diff
@@ -110,6 +110,19 @@ async function transformModule(entryPath: string, entry: TransformEntry) {
     });
   }
 
+  const resolveOptions: ResolveOptions = {
+    from: pathToFileURL(entryPath),
+    ...entry.resolve,
```

Comment:
i think we don't want to override `from` in particular (it depends on each module context). We might extend it though as fallback search path but probably not useful here (because resolver is mainly to resolve relative files correctly)

Reply:
Ok, I'll omit that from the input type also.